node/src/components/consensus/highway_core/finality_detector.rs

```diff
@@ -72,6 +72,10 @@ impl<C: Context> FinalityDetector<C> {
                 None
             };
 
+            trace!(
+                %block.height,
```

Comment:
Unfortunately this will only log the _relative_ height (since the beginning of this era).
Let's rather log [`finalized_block.height()` here](https://github.com/CasperLabs/casper-node/blob/360945360e3ec6e3682b71c988c74fa8e3923e46/node/src/components/consensus/era_supervisor.rs#L655).

Reply:
> I thought that NDRS-468 will measure how long it took to calculate the fork choice.

Yeah, I got confused with the details in the story, it is logging the time taken for `fork_choice` to complete